src/config/config.js

```diff
@@ -424,7 +424,7 @@ export let config = {
 
     get paypalDomains() : Object {
         return {
-            [ ENV.LOCAL ]:      `http://localhost.paypal.com:${ config.ports.default }`,
+            [ ENV.LOCAL ]:      /^https?:\/\/.*\.paypal\.com:?\d*$/,
```

Comment:
Making this a looser check - sometimes while testing locally, you might need to switch between stage pages and localhost pages.